cumulusci/cumulusci.yml

```diff
@@ -348,6 +348,10 @@ tasks:
         description: Apply org settings from a scratch org definition file or dict
         class_path: cumulusci.tasks.salesforce.org_settings.DeployOrgSettings
         group: Salesforce DX
+    promote_package_version:
+        description: Promote a 2gp package so that it can be installed in a production org
+        class_path: cumulusci.tasks.salesforce.promote_2gp_package.PromotePackageVersion
```

Comment:
I tried running the task and got:

ModuleNotFoundError: No module named 'cumulusci.tasks.salesforce.promote_2gp_package'

Reply:
Ya, we might want the task to actually run 🤷‍♂️